docs/howtoguides/enable_anbox.md

```diff
@@ -0,0 +1,45 @@
+# How to enable Anbox Cloud
+
+```{important}
+[Anbox Cloud is supported on 20.04 and 22.04 releases](https://anbox-cloud.io/).
+```
+
+To use Anbox, you will need to enable it directly throught the Ubuntu Pro Client,
+which will install all of the necessary snaps and setup the APT sources needed for
+the service.
+
+```{important}
```

Comment:
Can we combine this admonition with the one at the top of the page? It really confused me to see two of them so close together right at the start of the guide.

Reply:
I moved it to another part of the doc. Let me know what you think